### PR TITLE
(SIMP-5479) Skip fix_errata_on on windows

### PR DESCRIPTION
**Pull request by jeefberkey**
SIMP-5479 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/rubygem-simp-beaker-helpers/pipelines/33486108

**Comment by jeefberkey** [comment]
:heavy_check_mark: https://gitlab.com/simp/rubygem-simp-beaker-helpers/pipelines/33510050

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@
   c.before :suite do
     begin
       # Install modules and dependencies from spec/fixtures/modules
-      copy_fixture_modules_to( hosts )
+      # copy_fixture_modules_to( hosts )
```
Comment (op-ct) [inline spec/spec_helper_acceptance.rb]: Did you mean to leave this commented out?

File: spec/lib/simp/beaker_helpers_spec.rb [hunk 2]
```diff
@@ -80,11 +80,11 @@ def initialize
 
       # this logic won't work properly without code changes that just aren't worth it because
       # Puppet 4 is MD soon....
-      it "maps to appropriate Puppet version when '<' operator specified in version" do
-        pending 'fails because matches 4.x table'
-        allow(@helper).to receive(:`).with('gem search -ra -e puppet').and_return(gem_search_results)
-        expect( @helper.latest_puppet_agent_version_for('< 5.5') ).to match /5.4.0/
-      end
+      # it "maps to appropriate Puppet version when '<' operator specified in version" do
```
Comment (op-ct) [inline spec/lib/simp/beaker_helpers_spec.rb]: Did you mean to leave this commented out?

File: spec/acceptance/nodesets/default.yml [hunk 3]
```diff
@@ -1,22 +1,32 @@
...
+    box: centos/6
     hypervisor: vagrant
 
 CONFIG:
   log_level: verbose
-  type:      aio
+  type: aio
   vagrant_memsize: 256
+<% if ENV['BEAKER_PUPPET_ENVIRONMENT'] -%>
+  puppet_environment: <%= ENV['BEAKER_PUPPET_ENVIRONMENT'] %>
```
Comment (op-ct) [inline spec/acceptance/nodesets/default.yml]: * `BEAKER_PUPPET_ENVIRONMENT` should be `BEAKER_PUPPET_COLLECTION`
* `puppet_environment` should be `puppet_collection`

See [SIMP-5430][SIMP-5430] and pupmod-simp-tlog's [`.gitlab-ci.yml`][0]

[0]: https://github.com/simp/pupmod-simp-tlog/blob/master/.gitlab-ci.yml#L72-L91
[SIMP-5430]: https://simp-project.atlassian.net/browse/SIMP-5430

File: lib/simp/beaker_helpers.rb [hunk 4]
```diff
@@ -226,7 +226,13 @@ def enable_fips_mode_on( suts = hosts )
     puts '== configuring FIPS mode on SUTs'
     puts '  -- (use BEAKER_fips=no to disable)'
     parallel = (ENV['BEAKER_SIMP_parallel'] == 'yes')
+
     block_on(suts, :run_in_parallel => parallel) do |sut|
+      if sut[:platform] =~ /windows/
```
Comment (op-ct) [inline lib/simp/beaker_helpers.rb]: If `simp-beaker-helpers` supports Windows, we should include a Windows nodeset.

File: spec/spec_helper_acceptance.rb (hunk 1 above)
Comment (jeefberkey) [reply spec/spec_helper_acceptance.rb]: No, I'll delete the rest of it

File: spec/lib/simp/beaker_helpers_spec.rb (hunk 2 above)
Comment (jeefberkey) [reply spec/lib/simp/beaker_helpers_spec.rb]: Yes, and the comment is still relevant. Setting it to pending is no longer sufficient, however. At least on my system

File: spec/acceptance/nodesets/default.yml (hunk 3 above)
Comment (jeefberkey) [reply spec/acceptance/nodesets/default.yml]: Will fix

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/rubygem-simp-beaker-helpers/pipelines/33605405

File: lib/simp/beaker_helpers.rb (hunk 4 above)
Comment (op-ct) [reply lib/simp/beaker_helpers.rb]: :ballot_box_with_check: d8eaaf8

File: .gitlab-ci.yml [hunk 5]
```diff
@@ -1,120 +1,166 @@
...
+.spec_tests: &spec_tests
+  script:
+    - bundle exec rake spec
+
+# To avoid running a prohibitive number of tests every commit,
+# don't set this env var in your gitlab instance
+.only_with_SIMP_FULL_MATRIX: &only_with_SIMP_FULL_MATRIX
+  only:
+    variables:
+      - $SIMP_FULL_MATRIX
```
[the remark this answers is not in the document]
Comment (jeefberkey) [reply .gitlab-ci.yml]: This variable is already in many, many gitlab ci yamls.

[the remark this answers is not in the document]
Comment (trevor-vaughan) [reply .gitlab-ci.yml]: @jeefberkey Is this an issue with the `yes`/`no` part or the renaming part, or both?

**Pull request merged**